server/settings/imageio.py

```diff
@@ -63,9 +70,9 @@ def ensure_unique_names(cls, value):
 
 
 class OverrideNodesModel(NodesModel):
-    subsets: list[str] = SettingsField(
+    products: list[str] = SettingsField(
```

Comment:
This should probably be `Product names`?

Reply:
BTW is there automatic conversion of this change?

Reply:
No settings conversion was not here yet, merge happened to quickly. There's this now though: https://github.com/ynput/ayon-nuke/pull/88